qa/workunits/fs/misc/osd_full_mgr_hang.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -x
+
+set -e
+
+ceph fs subvolume create cephfs sub_0
+subvol_path=$(ceph fs subvolume getpath cephfs sub_0 2>/dev/null)
+
+#For debugging
+echo "Before write"
+df -h
+ceph osd df
+
+sudo dd if=/dev/urandom of=$CEPH_MNT$subvol_path/500MB_file-1 status=progress bs=1M count=500
+
+#To reflect in osd df output
+sleep 5
+
+#For debugging
+echo "After write"
+df -h
+ceph osd df
+
+ceph osd set-full-ratio 0.2
+ceph osd set-nearfull-ratio 0.16
+ceph osd set-backfillfull-ratio 0.18
+
+#For debugging
+echo "After ratio set"
+df -h
+ceph osd df
+
```

Comment:
You should confirm the pools are full. You could grep the output of `ceph health detail`. Instead of sleeping for static times (which is racy), it'd be better to poll for the condition you're waiting for with some timeout.

Reply:
Done